### ISSUE TITLE
not working with moodle3.8 while returning. it showing the blank page This page isn’t working  http://......... is currently unable to handle this request. HTTP ERROR 500

### ISSUE THREAD

**Issue by dhascruz** [Paytm-Payments/Paytm_Moodle_Kit#5]

**Comment by sangeeyeah** [comment]
The Paytm Moodle plugin works fine in 3.7. In 3.8, it goes up to the **http://website.com/enrol/paytm/itn.php** and gives a 500 error. 

--------------------------
This page isn’t working
website.com is currently unable to handle this request.
HTTP ERROR 500
--------------------------

Looks like some minor changes are required in the **itn.php** file

**Comment by dhascruz** [comment]
please advice me the what the changes need to be done. i can do it in our side

**Comment by sangeeyeah** [comment]
@Prabhat1Mishra please look into this plugin issue. Looks like an update is required in the **itn.php** file since it is generating the 500 error.

**Comment by dhascruz** [comment]
@Prabhat1Mishra  i am waiting for your response. please revert back ASAP

**Comment by Prabhat1Mishra** [comment]
Hello everyone,

You can raise your query to "integration.dev@paytm.com" with an error response for fast resolution.

**Comment by Prabhat1Mishra** [comment]
Hello everyone,

You can use our updated plugin.

**Comment by dhascruz** [comment]
Great Job. Please confirm what you done.

Do you done update the plugin.

Thank you very much
Thanks
Dhas

**Comment by Prabhat1Mishra** [comment]
Yes, some require changes are done in our updated plugin.
Please go throw with our updated plugin and confirm.

**Comment by gsuid** [comment]
> Hello everyone,
> 
> You can use our updated plugin.

Hi Prabhat,
I have used this updated plugin in my moodle site and this is working perfect. I have tested this with few UPI transactions. Thanks for your kind support. 
I also wants to know that whether this plugin is only supporting UPI payments?? Because I have seen only UPI payment option in my transaction page not others payment options like: Credit Card/Debit Card/Net Banking, etc ??
